packages/vsce/package.json

```
@@ -453,6 +458,11 @@
           "command": "cics-extension-for-zowe.showRegionAttributes",
           "group": ""
         },
+        {
+          "when": "view == cics-view && viewItem =~ /cicsregion.*\\.active/",
```

Could do this if you don't want to be able to select multiple? It's what we do with manage profile.

```suggestion
          "when": "view == cics-view && viewItem =~ /cicsregion.*\\.active/ && !listMultiSelection",
```